### ISSUE TITLE
[Fixed in GH-6052] Safari Firefox: Missing Visual Focus on Checkout: Shipping Address / Add New Address

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4320]
Disabled steps shouldn't receive focus before they're active as shown in the first screenshot

![Screen Shot 2019-08-29 at 3 14 23 PM](https://user-images.githubusercontent.com/15113219/63969531-088b3700-ca70-11e9-8d2d-3dede2f9c7c2.png)
![Screen Shot 2019-08-29 at 3 15 06 PM](https://user-images.githubusercontent.com/15113219/63969537-0c1ebe00-ca70-11e9-84ac-9a9cf558db9b.png)

**Comment by Zeyber** [comment]
QA: Tabbing through form should show focus on each applicable element. Should not be able to tab to active or disabled steps in checkout progress steps (check this works in other checkout steps).

**Comment by KateChuen** [comment]
Chrome: OK

Missing keyboard accessibility and visual focus on these :
- Set as default checkbox on Safari
- Back to Cart & Continue buttons on Safari
- active checkout steps on Firefox and Safari (although the comment said i should not be able to. But I would expect to be able to. And it works on Chrome, able to tab to active checkout steps).

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.


**Fixed:** Set as default checkbox on Safari

![Screen Shot 2020-01-30 at 11 16 02 AM](https://user-images.githubusercontent.com/15113219/73468572-fda43d80-4352-11ea-83ff-02976b077274.png)

**Fixed:** Back to Cart & Continue buttons on Safari

![Screen Shot 2020-01-30 at 11 20 56 AM](https://user-images.githubusercontent.com/15113219/73468647-20365680-4353-11ea-80de-760934442fe9.png)
![Screen Shot 2020-01-30 at 11 21 05 AM](https://user-images.githubusercontent.com/15113219/73468635-1876b200-4353-11ea-85a0-8a1587f16e6a.png)

**Fixed:** Active checkout steps on Firefox and Safari

![Screen Shot 2020-01-30 at 11 20 28 AM](https://user-images.githubusercontent.com/15113219/73468755-48be5080-4353-11ea-9264-e532b70035e7.png)
![Screen Shot 2020-01-30 at 11 22 38 AM](https://user-images.githubusercontent.com/15113219/73468765-4b20aa80-4353-11ea-952a-5ccacea73a4c.png)

**Comment by KateChuen** [comment]
Tested on dev3, on chrome, firefox, safari.
